src/core/server/saved_objects/permission_control/acl.test.ts

```diff
@@ -0,0 +1,261 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { Principals, Permissions, ACL } from './acl';
+
+describe('SavedObjectTypeRegistry', () => {
```

Comment:
should we have a name more related to ACL instead of `SavedObjectTypeRegistry`?

Reply:
Good catch, thanks!